includes/notes/class-wc-payments-notes-instant-deposits-eligible.php

```diff
@@ -23,19 +23,18 @@ class WC_Payments_Notes_Instant_Deposits_Eligible {
 	/**
 	 * Name of the note for use in the database.
 	 */
-	// TODO: Get the proper doc url.
-	const NOTE_DOCUMENTATION_URL = '';
+	const NOTE_DOCUMENTATION_URL = 'https://docs.woocommerce.com/document/payments/';
 
 	/**
 	 * Get the note.
 	 */
 	public static function get_note() {
-		$note_class = WC_Payment_Woo_Compat_Utils::get_note_class();
-		$note       = new $note_class();
+		$note_class   = WC_Payment_Woo_Compat_Utils::get_note_class();
+		$note         = new $note_class();
+		$note_content = __( 'You are now eligible for Instant Deposits! Skip the wait and get paid instantly.', 'woocommerce-payments' );
 
-		// TODO: get the proper title and message.
 		$note->set_title( __( 'Instant deposits available', 'woocommerce-payments' ) );
-		$note->set_content( __( 'You are now eligible for Instant Deposits!', 'woocommerce-payments' ) );
+		$note->set_content( $note_content );
```

Comment:
Little nitpick 🙁  You can skip `$note_content` variable and write it directly in `$note->set_content`.

Reply:
😀  That's how I initially had it, then moved it due to it was getting a little longer and I wasn't sure if it would grow beyond that. I'll put it back for now.